pom.xml

```diff
@@ -244,7 +244,7 @@
         <jts.spatial4j.version>1.15.0</jts.spatial4j.version>
         <jts.version>1.14.0</jts.version>
         <jvnet-ogc.version>2.6.1</jvnet-ogc.version>
-        <karaf.version>4.2.2</karaf.version>
+        <karaf.version>4.2.3</karaf.version>
```

Comment:
✏️ Update the versions of Karaf in the docs:
https://github.com/codice/ddf/blob/master/distribution/docs/src/main/resources/content/_reference/_dependencyList/ddf-dependency-list.adoc

Reply:
this PR will automate this https://github.com/codice/ddf/pull/4986